i18n/tooltips/en.ts

```diff
@@ -41,10 +41,10 @@ const en: TooltipTranslations = {
           </p> \
           <ul style='list-style-type: disc; padding-bottom: 8px; padding-left: 20px;'> \
             <li>a Canadian citizen</li> \
+            <li>an Indigenous person registered under the <em>Indian Act</em></li> \
```

Comment:
if this was changed for political reasons. indigenous  persons were here first, they should be first

Reply:
Very true lol